src/v2/ion/i18nTransformer.js

```diff
@@ -98,7 +98,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'authenticator-enrollment-data.phone_number.authenticator.methodType.voice': 'oie.phone.enroll.voice.label',
 
   'enroll-authenticator.okta_password.credentials.passcode': 'oie.password.passwordLabel',
-  'enroll-authenticator.okta_email.credentials.passcode': 'email.enroll.enterCode',
+  'enroll-authenticator.okta_email.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
```

Comment:
String within brackets is customized `Enter Code`

Before: 
![Screen Shot 2021-05-26 at 5 07 02 PM](https://user-images.githubusercontent.com/23267876/119746239-da08de80-be44-11eb-904d-76673ddde963.png)




After: 

![Screen Shot 2021-05-26 at 5 03 35 PM](https://user-images.githubusercontent.com/23267876/119746048-5f3fc380-be44-11eb-9d25-e2be01fea3f1.png)